pandas/io/excel/_base.py

```diff
@@ -667,6 +667,17 @@ class ExcelWriter(metaclass=abc.ABCMeta):
         be parsed by ``fsspec``, e.g., starting "s3://", "gcs://".
 
         .. versionadded:: 1.2.0
+    if_sheet_exists : {'new', 'replace', 'overwrite', 'fail'}, default 'new'
```

Comment:
why is the default not 'error'?

Reply:
Mainly to preserve the current behaviour as the default. Happy to change